### PR TITLE
ngtcp2: switch to apple-sdk_11

### LINKED ISSUE #354435: Build failure: ngtcp2
### Steps To Reproduce

1. build ngtcp2

### Build log

<details>
<summary>Details</summary>

```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/scns4a3xhpws9j4440azjx1z621lyh1c-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LOCALEDIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/nn822xiv1w67db3v25a63h0pva60zwd8-ngtcp2-1.8.0-doc/share/doc/ngtcp2 -DCMAKE_INSTALL_INFODIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/2m82xhsvr5s09f2hb3cf8ayw6ppfd5wg-ngtcp2-1.8.0-dev/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/2m82xhsvr5s09f2hb3cf8ayw6ppfd5wg-ngtcp2-1.8.0-dev/include -DCMAKE_INSTALL_SBINDIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/strip -DCMAKE_RANLIB=/nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/ranlib -DCMAKE_AR=/nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0 -DENABLE_STATIC_LIB:BOOL=FALSE
-- The C compiler identification is Clang 16.0.6
-- The CXX compiler identification is Clang 16.0.6
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenSSL: /nix/store/wjf40mvkm1vg4ndribs4ds5kavh7rdfq-quictls-3.3.0-quic1/lib/libcrypto.dylib (found suitable version "3.3.0", minimum required is "1.1.1")
-- Found Libev: /nix/store/zxmn3l312l1iksk5wvwi8qp5manpmjvk-libev-4.33/lib/libev.dylib (found suitable version "4.33", minimum required is "4.11")
-- Found Libnghttp3: /nix/store/vhij8rw82h5jy34xv9bdyvy17p7lqhd2-nghttp3-1.6.0/lib/libnghttp3.dylib (found suitable version "1.6.0", minimum required is "1.0.0")
-- Found Libbrotlienc: /nix/store/75zlljjagbr7jnalvbl1dlm5aa29xi1w-brotli-1.1.0-lib/lib/libbrotlienc.dylib (Required is at least version "1.0.9")
-- Found Libbrotlidec: /nix/store/75zlljjagbr7jnalvbl1dlm5aa29xi1w-brotli-1.1.0-lib/lib/libbrotlidec.dylib (Required is at least version "1.0.9")
-- Looking for SSL_provide_quic_data
-- Looking for SSL_provide_quic_data - found
-- Looking for arpa/inet.h
-- Looking for arpa/inet.h - found
-- Looking for netinet/in.h
-- Looking for netinet/in.h - found
-- Looking for netinet/ip.h
-- Looking for netinet/ip.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for sys/endian.h
-- Looking for sys/endian.h - not found
-- Looking for endian.h
-- Looking for endian.h - not found
-- Looking for byteswap.h
-- Looking for byteswap.h - not found
-- Looking for asm/types.h
-- Looking for asm/types.h - not found
-- Looking for linux/netlink.h
-- Looking for linux/netlink.h - not found
-- Looking for linux/rtnetlink.h
-- Looking for linux/rtnetlink.h - not found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of ssize_t
-- Check size of ssize_t - done
-- Looking for bswap_64
-- Looking for bswap_64 - not found
-- Looking for explicit_bzero
-- Looking for explicit_bzero - not found
-- Looking for memset_s
-- Looking for memset_s - found
-- summary of build options:

    Package version: 1.8.0
    Library version: 18:4:2
    Install prefix:  /nix/store/mcg8k4zshl24z3qw0jw7jyhljlafy1ma-ngtcp2-1.8.0
    Target system:   Darwin
    Compiler:
      Build type:     Release
      C compiler:     /nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/clang
      CFLAGS:         -O3
      C++ compiler:   /nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/clang++
      CXXFLAGS:       -O3
      WARNCFLAGS:       -Wextra -Wall -Wconversion -Winline -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wpointer-arith -Wshadow -Wundef -Wwrite-strings -Waddress -Wattributes -Wcast-align -Wdeclaration-after-statement -Wdiv-by-zero -Wempty-body -Wendif-labels -Wfloat-equal -Wformat-nonliteral -Wformat-security -Wmissing-field-initializers -Wmissing-noreturn -Wno-format-nonliteral -Wredundant-decls -Wsign-conversion -Wstrict-prototypes -Wunreachable-code -Wunused-macros -Wunused-parameter -Wvla -Wshorten-64-to-32 -Wlanguage-extension-token -Wpragmas -Wunreachable-code-break -Wheader-guard -Wsometimes-uninitialized -Wmissing-variable-declarations
      WARNCXXFLAGS:     -Wall -Wformat-security -Wsometimes-uninitialized -Wno-noexcept-type
    Library:
      Shared:         ON
      Static:         FALSE
    Test:
      Build Test:     OFF
    Libs:
      OpenSSL:        TRUE (LIBS='/nix/store/wjf40mvkm1vg4ndribs4ds5kavh7rdfq-quictls-3.3.0-quic1/lib/libssl.dylib;/nix/store/wjf40mvkm1vg4ndribs4ds5kavh7rdfq-quictls-3.3.0-quic1/lib/libcrypto.dylib')
      Libev:          TRUE (LIBS='/nix/store/zxmn3l312l1iksk5wvwi8qp5manpmjvk-libev-4.33/lib/libev.dylib')
      Libnghttp3:     TRUE (LIBS='/nix/store/vhij8rw82h5jy34xv9bdyvy17p7lqhd2-nghttp3-1.6.0/lib/libnghttp3.dylib')
      GnuTLS:         FALSE (LIBS='')
      BoringSSL:      FALSE (LIBS='')
      Picotls:        FALSE (LIBS='')
      wolfSSL:        FALSE (LIBS='')
      Jemalloc:        (LIBS='')
      Libbrotlienc:   TRUE (LIBS='/nix/store/75zlljjagbr7jnalvbl1dlm5aa29xi1w-brotli-1.1.0-lib/lib/libbrotlienc.dylib')
      Libbrotlidec:   TRUE (LIBS='/nix/store/75zlljjagbr7jnalvbl1dlm5aa29xi1w-brotli-1.1.0-lib/lib/libbrotlidec.dylib')

-- Configuring done (42.4s)
-- Generating done (0.3s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /tmp/nix-build-ngtcp2-1.8.0.drv-0/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
configurePhase completed in 44 seconds
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j12 SHELL=/nix/store/qwg7nqq0y2lzc94pdaa00li2z12h4jl4-bash-5.2p37/bin/bash
[  1%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_pkt.c.o
[  3%] Building C object third-party/CMakeFiles/http-parser.dir/http-parser/http_parser.c.o
[  6%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_conn.c.o
[  6%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_conv.c.o
[  7%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_str.c.o
[ 12%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_buf.c.o
[ 12%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_pq.c.o
[ 12%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_vec.c.o
[ 13%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_map.c.o
[ 15%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_mem.c.o
[ 16%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_rob.c.o
[ 18%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_ppe.c.o
[ 19%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_crypto.c.o
[ 21%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_err.c.o
[ 22%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_range.c.o
[ 24%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_acktr.c.o
[ 25%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_rtb.c.o
[ 27%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_frame_chain.c.o
[ 28%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_idtr.c.o
[ 30%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_strm.c.o
[ 31%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_gaptr.c.o
[ 33%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_ringbuf.c.o
[ 33%] Built target http-parser
[ 34%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_log.c.o
[ 36%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_qlog.c.o
[ 37%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_ksl.c.o
[ 39%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_cid.c.o
[ 40%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_cc.c.o
[ 42%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_bbr.c.o
[ 43%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_addr.c.o
[ 45%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_path.c.o
[ 46%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_pv.c.o
[ 48%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_pmtud.c.o
[ 50%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_version.c.o
[ 51%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_rst.c.o
[ 53%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_window_filter.c.o
[ 54%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_opl.c.o
[ 56%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_balloc.c.o
[ 57%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_objalloc.c.o
[ 59%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_unreachable.c.o
[ 60%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_transport_params.c.o
[ 62%] Building C object lib/CMakeFiles/ngtcp2.dir/ngtcp2_settings.c.o
[ 63%] Linking C shared library libngtcp2.dylib
[ 63%] Built target ngtcp2
[ 66%] Building C object crypto/quictls/CMakeFiles/ngtcp2_crypto_quictls.dir/__/shared.c.o
[ 66%] Building C object crypto/quictls/CMakeFiles/ngtcp2_crypto_quictls.dir/quictls.c.o
[ 68%] Linking C shared library libngtcp2_crypto_quictls.dylib
[ 68%] Built target ngtcp2_crypto_quictls
[ 69%] Building CXX object examples/CMakeFiles/qtlsclient.dir/client.cc.o
[ 71%] Building CXX object examples/CMakeFiles/qtlsserver.dir/server.cc.o
[ 75%] Building CXX object examples/CMakeFiles/qtlsclient.dir/debug.cc.o
[ 75%] Building CXX object examples/CMakeFiles/qtlsserver.dir/debug.cc.o
[ 77%] Building CXX object examples/CMakeFiles/qtlsclient.dir/util.cc.o
[ 78%] Building CXX object examples/CMakeFiles/qtlsclient.dir/shared.cc.o
[ 78%] Building CXX object examples/CMakeFiles/qtlsserver.dir/util.cc.o
[ 80%] Building CXX object examples/CMakeFiles/qtlsclient.dir/client_base.cc.o
[ 81%] Building CXX object examples/CMakeFiles/qtlsserver.dir/server_base.cc.o
[ 84%] Building CXX object examples/CMakeFiles/qtlsserver.dir/http.cc.o
[ 84%] Building CXX object examples/CMakeFiles/qtlsclient.dir/tls_client_context_quictls.cc.o
[ 86%] Building CXX object examples/CMakeFiles/qtlsserver.dir/shared.cc.o
[ 87%] Building CXX object examples/CMakeFiles/qtlsclient.dir/tls_client_session_quictls.cc.o
/tmp/nix-build-ngtcp2-1.8.0.drv-0/source/examples/shared.cc:63:30: /tmp/nix-build-ngtcp2-1.8.0.drv-0/source/examples/shared.cc:error: 63:30: error: use of undeclared identifier 'IP_RECVTOS'
          cmsg->cmsg_type == IP_RECVTOS
                             ^
use of undeclared identifier 'IP_RECVTOS'
          cmsg->cmsg_type == IP_RECVTOS
                             ^
/tmp/nix-build-ngtcp2-1.8.0.drv-0/source/examples/shared.cc:93:36: error: use of undeclared identifier 'IP_RECVTOS'
    if (setsockopt(fd, IPPROTO_IP, IP_RECVTOS, &tos,
                                   ^
/tmp/nix-build-ngtcp2-1.8.0.drv-0/source/examples/shared.cc:93:36: error: use of undeclared identifier 'IP_RECVTOS'
    if (setsockopt(fd, IPPROTO_IP, IP_RECVTOS, &tos,
                                   ^
2 errors generated.
2 errors generated.
make[2]: *** [examples/CMakeFiles/qtlsserver.dir/build.make:146：examples/CMakeFiles/qtlsserver.dir/shared.cc.o] 错误 1
make[2]: *** 正在等待未完成的任务....
make[2]: *** [examples/CMakeFiles/qtlsclient.dir/build.make:132：examples/CMakeFiles/qtlsclient.dir/shared.cc.o] 错误 1
make[2]: *** 正在等待未完成的任务....
[ 89%] Building CXX object examples/CMakeFiles/qtlsclient.dir/tls_session_base_quictls.cc.o
make[1]: *** [CMakeFiles/Makefile2:311：examples/CMakeFiles/qtlsclient.dir/all] 错误 2
make[1]: *** 正在等待未完成的任务....
make[1]: *** [CMakeFiles/Makefile2:339：examples/CMakeFiles/qtlsserver.dir/all] 错误 2
make: *** [Makefile:146：all] 错误 2
```

</details>

### Notify maintainers

@izorkin

### Metadata

 - system: `"x86_64-darwin"`
 - host os: `Darwin 24.1.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.24.9`
 - nixpkgs: `/nix/store/wb6agba4kfsxpbnb5hzlq58vkjzvbsk6-source`

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by XYenon**
Fix https://github.com/NixOS/nixpkgs/issues/354435

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by azuwis — CHANGES_REQUESTED** [review]
According to https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/platform-notes.chapter.md

> There are multiple versions of the SDK packages in Nixpkgs, but one is included by default in the stdenv. Usually, you don’t have to change or pick a different SDK. When in doubt, use the default.

> When determining whether to use a non-default SDK, consider the following:
>
> * Try building your derivation with the default SDK. If it works, you’re done.
> * If the package specifies a specific version, use that. See below for how to map Xcode version to SDK version.
> * If the package’s documentation indicates it supports optional features on newer SDKs, consider using the SDK that enables those features. If you’re not sure, use the default SDK.


Upstream does not require a specific apple SDK version, should use the default `apple-sdk`.

I've verified that it builds fine with default apple sdk on aarch64-darwin.

File: pkgs/development/libraries/ngtcp2/default.nix [hunk 1]
```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchFromGitHub
 , cmake
 , brotli, libev, nghttp3, quictls
-, CoreServices
+, apple-sdk_15
```
[the remark this answers is not in the document]
Comment (XYenon) [reply pkgs/development/libraries/ngtcp2/default.nix:4]: Build failed on x86_64-darwin with the same error of https://github.com/NixOS/nixpkgs/issues/354435.

[the remark this answers is not in the document]
Comment (XYenon) [reply pkgs/development/libraries/ngtcp2/default.nix:4]: But it can be built with apple-sdk_11.

**Review by azuwis — APPROVED** [review]

**Review by Izorkin — APPROVED** [review]

**Review by khaneliman — APPROVED** [review]
LGTM, I don't mind leaving it in a condition right now to keep rebuilds down.

**Comment by khaneliman** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 355344`

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>curlHTTP3</li>
    <li>curlHTTP3.bin</li>
    <li>curlHTTP3.dev</li>
    <li>curlHTTP3.devdoc</li>
    <li>curlHTTP3.man</li>
    <li>ngtcp2</li>
    <li>ngtcp2.dev</li>
    <li>ngtcp2.doc</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>curlHTTP3</li>
    <li>curlHTTP3.bin</li>
    <li>curlHTTP3.dev</li>
    <li>curlHTTP3.devdoc</li>
    <li>curlHTTP3.man</li>
    <li>ngtcp2</li>
    <li>ngtcp2.dev</li>
    <li>ngtcp2.doc</li>
  </ul>
</details>

**Pull request merged**